docs/dev-guide/contributing/branching.rst

```diff
@@ -17,7 +17,7 @@ on the `Semantic Versioning <http://semver.org/>`_ strategy.
 
 .. _PUP-0003: https://github.com/pulp/pups/blob/master/pup-0003.md
 
-master
+2-master
 ------
```

Comment:
Increase the length of the "underline" by 2, so that it's the same length as the heading.  It causes a docs build error